src/Core/Settings/GlobalSettings.cs

```diff
@@ -310,6 +310,12 @@ public class EventLoggingSettings
         public AzureServiceBusSettings AzureServiceBus { get; set; } = new AzureServiceBusSettings();
         public RabbitMqSettings RabbitMq { get; set; } = new RabbitMqSettings();
         public int IntegrationCacheRefreshIntervalMinutes { get; set; } = 10;
+        public int UserCacheTtlMinutes { get; set; } = 60;
```

Comment:
🎨 Per my other comment, if this is generic perhaps we also rethink how this is loaded and defined.